src/api-reference/common/event-pubsub/travel-search-event.markdown

```diff
@@ -107,7 +107,8 @@ Name|Type|Format|Description
 Name|Type|Format|Description
 ---|---|---|---
 `companyId`|`String`|GUID|Uniquely identifies the company of the traveler.
-`userId`|`String`|GUID|Uniquely identifies the user performing the search. *NOTE:* In the event travel is booked by an arranger, this is the traveler. In cases where the individual is booking on behalf of a guest, this is the user performing the search.
+`userId`|`String`|GUID|Uniquely identifies the user performing the search. *NOTE:* In cases where the individual is booking on behalf of a guest, this is the user performing the search.
```

Comment:
userId uniquely identifies the traveler. it is not necessarily the user who is performing the search. in a case of an arranger booking it would be the arrangerUserId who is actually performing the search.

Reply:
looks good. Just added "e.g. arranger" to make it even more direct.